src/book/01-team/05-robocup/03-debriefs.mdx

```diff
@@ -10,6 +10,146 @@ authors:
   - Lachlan Court (@LachlanCourt)
 ---
 
+## 2023: Bordeaux, France
+
+### What went wrong
```

Comment:
I think we should swap the order "wrong → well → next" to "well → wrong → next". Start with the good news! 😄

On this and the other briefings.

Reply:
but you want to end on a good note and then optimism!

Reply:
Hmmm fair point. Trying to avoid the [Anchoring bias](https://www.scribbr.com/research-bias/anchoring-bias/#:~:text=Revised%20on%20June%202%2C%202023,anchor%2C%20to%20make%20subsequent%20judgments.), where the first thing you see unduly colors everything else that follows.